gitgud/operations.py

```diff
@@ -43,6 +43,10 @@ def add_and_commit(self, name):
 
         return commit
 
+    def show_tree(self):
+        tree = self.repo.git.log('--graph', '--oneline', '--all', '--decorate=short')
```

Comment:
From https://git-scm.com/docs/git-log:

The default option for --decorate is short, so we can remove it.

Also, I think show_tree implies that it'll be the one showing it, so I think it should handle that. The quick solution to that would be to move the print statement there, but......

Git itself shows pretty colors when running that command. Can you see if you can avoid using a print statement and have git do the printing so it keeps all the pretty colors?

Reply:
I'll get rid of =short, but I'm going to have to leave "--decorate"; otherwise, the references won't show up (unlike manually typing "git log --graph --oneline --all" where '--decorate' is implied for some reason) 

Yeah, I originally thought if I left out the print command, it would execute as if I had wrote "git log etc." But, if I don't turn it into a string and print it (just run "self.file_operator.repo.git.log(<flags>)) nothing will happen. There will be a short delay from some processing, but nothing will print to console.